src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts

```diff
@@ -2526,24 +2530,24 @@ export class NotebookEditorWidget extends Disposable implements INotebookEditorD
 		return ret;
 	}
 
-	async highlightFind(matchIndex: number): Promise<number> {
+	async highlightFind(matchIndex: number, ownerID?: string): Promise<number> {
 		if (!this._webview) {
 			return 0;
 		}
 
-		return this._webview?.findHighlight(matchIndex);
+		return this._webview?.findHighlightCurrent(matchIndex, ownerID ?? this.getId());
 	}
 
-	async unHighlightFind(matchIndex: number): Promise<void> {
+	async unHighlightFind(matchIndex: number, ownerID?: string): Promise<void> {
```

Comment:
Maybe we could also align the naming with webview, `findUnHighlightCurrent` is much better